packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -0,0 +1,286 @@
+//tslint:disable:no-big-function no-commented-code completed-docs
+import { Component, DebugElement, NO_ERRORS_SCHEMA, ViewChild } from "@angular/core";
+import { ComponentFixture, TestBed, async } from "@angular/core/testing";
+import { MatOptionModule, MatSelectModule } from "@angular/material";
+import { STARK_LOGGING_SERVICE } from "@nationalbankbelgium/stark-core";
+import { StarkDropdownComponent } from "./dropdown.component";
+import { CommonModule } from "@angular/common";
+import { By } from "@angular/platform-browser";
+import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
+import { TranslateModule, TranslateService } from "@ngx-translate/core";
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```

Comment:
Normally for unit tests, we should use the `NoopAnimationsModule` to make the tests lighter so no animations are triggered.